detections/endpoint/windows_create_test_registry.yml

```
@@ -0,0 +1,67 @@
+name: Windows Create Test Registry
+id: 80402396-d78a-4c6e-ade5-7697ea670adf
+version: 1
+date: '2025-02-11'
+author: Teoderick Contreras, Splunk
+status: production
+type: TTP
+description: The following analytic detects modifications to the Windows registry under `SOFTWARE\Microsoft\Test`, a location rarely used by legitimate applications in a production environment. Monitoring this key is crucial, as adversaries may create or alter values here for persistence, privilege escalation, or system manipulation. The detection leverages **Sysmon Event ID 13** (Registry Value Set) to identify unauthorized changes. Analysts should investigate processes associated with these modifications, particularly unsigned executables or suspicious command-line activity, as they may indicate malware or unauthorized software behavior.
```

The description does not explain why is the registry key important or interesting. It states generically that it could be used for persistence, etc. But this is true for almost any custom registry key.

Can you elaborate on why this was used, and if its specific to a TA or malware please specify.

```suggestion
description: The following analytic detects modifications to the Windows registry under `SOFTWARE\Microsoft\Test`, a location rarely used by legitimate applications in a production environment. Monitoring this key is crucial, as adversaries may create or alter values here for persistence, privilege escalation, or system manipulation. The detection leverages **Sysmon Event ID 13** (Registry Value Set) to identify unauthorized changes. Analysts should investigate processes associated with these modifications, particularly unsigned executables or suspicious command-line activity, as they may indicate malware or unauthorized software behavior.
```